### ISSUE TITLE
Bug in Intellij IDEA plugin, multiple messages at once

### ISSUE THREAD

**Issue by ellet0** [pinterest/ktlint#2659]
## Observed Behavior
The Intellij IDEA [Ktlint](https://plugins.jetbrains.com/plugin/15057-ktlint) plugin show the message `Ktlint found <number> lint violations of which <number> can be autocorrected>`

![image](https://github.com/pinterest/ktlint/assets/73608287/d3b47f82-5a6f-4713-bc20-1263ea5e5194)

## Expected Behavior

It would be useful to show the message only once or add an option to close the message as showing it multiple times can cover the whole editor or even the screen and it keep going infinitely

## Steps to Reproduce

This issue report is not very useful because I'm not sure how to reproduce it, I mostly focus on the code, not the message, but there is one way that might work which is to cause too many lint violations in the project (around 25 or more) and then once the message show keep coding and ignore it and it might duplicated more

## Your Environment

The Ktlint mode is not set to any option in the project but I think it did happen on another project where I actually have Ktlint enabled

![image](https://github.com/pinterest/ktlint/assets/73608287/92da0fa4-079e-450f-bfc5-dd848ff77569)

![image](https://github.com/pinterest/ktlint/assets/73608287/337ae09d-cb0c-48f1-bd6e-b3af67426697)

IntelliJ IDEA info:

```

IntelliJ IDEA 2024.1.1 (Community Edition)
Build #IC-241.15989.150, built on April 29, 2024
Runtime version: 17.0.10+1-b1207.14 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 14.2.1
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 10
Metal Rendering is ON
Registry:
  ide.experimental.ui=true
  terminal.new.ui=true
Non-Bundled Plugins:
  com.jetbrains.space (241.15989.9)
  com.nbadal.ktlint (0.22.0)
  com.varabyte.kobweb (0.1.0)
Kotlin: 241.15989.150-IJ


```

* Version of ktlint used: IDE plugin version is 0.22.0
* Relevant parts of the `.editorconfig` I didn't use it
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): for this project, I didn't enable Ktlint Gradle plugin or use it, the message kept showing in `.kt` and `.kts` files which is okay but it would be nice to have an option to close/ignore it or fixing that bug cause the message to be duplicated multiple times
* Version of Gradle used (if applicable): 8.7
* Operating System and version: macOS m1 (14.2.1 (23C71))

**Comment by paul-dingemans** [comment]
The ktlint-intellij-plugin lives at another [repository](https://github.com/nbadal/ktlint-intellij-plugin/issues). Please create the issue in that project. But I have already had a look an cannot reproduce it.
![Screenshot 2024-05-16 at 18 31 35](https://github.com/pinterest/ktlint/assets/5195292/9b479033-ba07-4b7b-9f8c-d3442e97d32c)

**Comment by ellet0** [comment]
> The ktlint-intellij-plugin lives at another [repository](https://github.com/nbadal/ktlint-intellij-plugin/issues). Please create the issue in that project. But I have already had a look an cannot reproduce it. ![Screenshot 2024-05-16 at 18 31 35](https://private-user-images.githubusercontent.com/5195292/331296097-9b479033-ba07-4b7b-9f8c-d3442e97d32c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4Nzc0ODgsIm5iZiI6MTcxNTg3NzE4OCwicGF0aCI6Ii81MTk1MjkyLzMzMTI5NjA5Ny05YjQ3OTAzMy1iYTA3LTRiN2ItOWY4Yy1kMzQ0MmU5N2QzMmMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUxNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MTZUMTYzMzA4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YTZiZTBkYzhhOWEyMzc3NTQyNjQ1NTY3NDgzNDM4NmQxZGI0OTJlYzA4ZDRkODlmYzFhMzk3MTM1MmY1N2JhYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.R8YyWidC9QbBr79gjIFg146F_mg5aAF0PjEVeEx_Piw)

it happens after some hours, not directly, still would be nice to have the close/ignore button
